### DISCUSSION TITLE
Where do you specify the SSH key when using the binary?

### DISCUSSION THREAD

**Discussion by beanman109** [henrygd/beszel#8] [General]
I keep getting slapped with "KEY environment variable is not set" when trying to setup the agent using the binary.
However I can't seem to figure out where you actually specify this & the instructions don't include much about this.
Running `./beszel-agent --KEY XXXXX` or` ./beszel-agent --KEY=XXXXX` doesn't seem to do anything.

**Comment by beanman109** [comment]
Okay I'm dumb it actually goes in your system env variables

**Comment by henrygd** [comment]
Just to add on in case anyone else ends up here, the easiest way to include it is to just put it in front of the command:

```
KEY="..." ./beszel-agent
```

And if running as a systemd service you can include it in the Service section of the unit file like this:

```
Environment="KEY=..."
```
